File: 217-contains-duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function (nums) {
  nums.sort();
  for (var n = 0; n < nums.length; n++) {
    if (n == nums.length - 1) {
      return false;
    } else if (nums[n] == nums[n + 1]) {
      return true;
    }
  }
};

File: 217_contains_duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        std::ranges::sort(nums);
        for (int i = 0; i < nums.size(); i++) {
            if (i == nums.size() - 1) {
                return false;
            } else if (nums[i] == nums[i + 1]) {
                return true;
            }
        }
        return false;
    }
};